src/main/scala/com/gu/emailservices/ContributionEmailFields.scala

```diff
@@ -0,0 +1,41 @@
+package com.gu.emailservices
+
+import com.gu.i18n.Currency
+import com.gu.support.workers.model.{DirectDebitPaymentMethod, PaymentMethod}
+import org.joda.time.DateTime
+
+case class ContributionEmailFields(
+    email: String,
+    created: DateTime,
+    amount: BigDecimal,
+    currency: Currency,
+    edition: String,
+    name: String,
+    paymentMethod: Option[PaymentMethod] = None,
+    directDebitMandateId: Option[String] = None
+) extends EmailFields {
+
+  val paymentFields = paymentMethod match {
+    case Some(dd: DirectDebitPaymentMethod) => List(
+      "account name" -> dd.bankTransferAccountName,
+      "account number" -> mask(dd.bankTransferAccountNumber),
+      "sort code" -> hyphenate(dd.bankCode),
+      "Mandate ID" -> directDebitMandateId.getOrElse(""),
+      "first payment date" -> formatDate(created.plusDays(10)),
```

Comment:
Is this correct for a recurring contribution? I thought we tried to take the payment immediately?

Reply:
It is right for DD, I can't remember exactly but I think it's to give time for the DD to be set up?

Reply:
I guess it makes sense that it would take a number of days for GoCardless to set up the mandate etc. Thanks for clarifying 👍